recipes/r-tftargets/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = 'tftargets' %}
+{% set version = '1.3' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - https://github.com/slowkow/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: e27ed65c283779b4dcda6e0704a37112a81bfedb198b3d7d036c1a80083670ca
+
+build:
```

Comment:
Packages that don't have anything in the `compiler` section should be `noarch`, so please add `noarch: generic` here under the `build` section. Please read more about this here, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#architecture-independent-packages.

Reply:
Got it. Thanks for the reference!